### PR TITLE
🎨  458 home page base UX changes

### PR DESCRIPTION
**Pull request by xoscar**
This PR updates the styling and UX for the home page to match the design changes.

## Changes

- New test card
- New Layout colors
- Adding a color theme
- Including storybook

## Fixes

- https://github.com/kubeshop/tracetest/issues/458

## Checklist

- [x] tested locally
- [x] added new dependencies
- [ ] updated the docs
- [x] added a test

https://www.loom.com/share/390ea6904ac84b6db521bbae3fdfb49d

### CODE REVIEW

File: web/.storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module.exports = {
```
Comment (xoscar) [inline web/.storybook/main.js:1]: Storybook predefined config

File: web/craco.config.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const CracoLessPlugin = require('craco-less');
+
+module.exports = {
```
Comment (xoscar) [inline web/craco.config.js:3]: Using [CRACO](https://github.com/gsoft-inc/craco) to overwrite the less variables

File: web/cypress/integration/Home/ShowTestList.spec.ts [hunk 3]
```diff
@@ -1,20 +1,37 @@
 import {DOCUMENTATION_URL, GITHUB_URL} from '../../../src/constants/Common.constants';
+import {createTest, deleteTest} from '../utils/common';
```
Comment (xoscar) [inline web/cypress/integration/Home/ShowTestList.spec.ts:2]: Adding more tests

File: web/src/components/CreateTestModal/CreateTestForm.tsx [hunk 4]
```diff
@@ -169,17 +169,20 @@ const CreateTestForm: React.FC<ICreateTestFormProps> = ({onSubmit, form}) => {
...
-                          if (fields.length === 1 || fields.length - 1 === index) {
-                            add();
-                          }
-                        }}
+                        onClick={() => remove(field.name)}
                       />
                     </Form.Item>
                   </div>
                 ))}
+                <Button
```
Comment (xoscar) [inline web/src/components/CreateTestModal/CreateTestForm.tsx:177]: Adding new add header button

File: web/src/components/TestCard/__stories__/TestCard.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import faker from '@faker-js/faker';
+import {ComponentStory, ComponentMeta} from '@storybook/react';
+import {HTTP_METHOD} from '../../../constants/Common.constants';
+
```
Comment (xoscar) [inline web/src/components/TestCard/__stories__/TestCard.stories.tsx:4]: Stories ✨

File: web/src/components/TestCard/TestCard.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
+import {RightOutlined} from '@ant-design/icons';
```
Comment (xoscar) [inline web/src/components/TestCard/TestCard.tsx:1]: New test card component 
Screen Shot 2022-05-10 at 10 58 49 a m

**Review by kdhamric — APPROVED** [review]
Watched the video (thanks!) - looks great.  Sorry it took me so long to review...

File: web/src/components/TestCard/__stories__/TestCard.stories.tsx (hunk 5 above)
Comment (cescoferraro) [reply web/src/components/TestCard/__stories__/TestCard.stories.tsx:4]: 🔥

**Review by cescoferraro — APPROVED** [review]
LGTM

**Pull request merged**